packages/design-system-scripts/gulp/common/constants.js

```diff
@@ -4,13 +4,17 @@ const CORE_SOURCE_PACKAGE = '@cmsgov/design-system';
 const CORE_DOCS_PACKAGE = '@cmsgov/design-system-docs';
 
 const REACT_DATA_DIR = `tmp/data`;
-const REACT_DATA_FILENAME = 'react-doc.json';
-const REACT_DATA_PATH = path.resolve(REACT_DATA_DIR, REACT_DATA_FILENAME);
+const REACT_PROP_DATA_FILENAME = 'react-prop.json';
```

Comment:
Split up the react data into 2 files, one for example code and one for prop data. This allows us to watch the two file types separately, and make `yarn start` slightly more efficient